net/netns.go

```diff
@@ -1,10 +1,8 @@
 package net
```

Comment:
I'd add `// +build go1.10` to prevent from accidentally compiling with older Go.